### PR TITLE
Molly facts

### PR DESCRIPTION
**Pull request by peeples**
Added a true Molly fact.

### CODE REVIEW

**Comment by peeples** [comment]
Fixed some things.

File: scan.py [hunk 1]
```diff
@@ -7,12 +7,13 @@ def scan(ints):
             total += val
             previous = val
         elif val < previous:
-            total = 0
+            print "a list of integers is not a giraffe"
+            total = 1
```
Comment (embray) [inline scan.py]: This is really annoying, don't do this.

File: erik.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+Erik also does not have blue hair.
```
Comment (embray) [inline erik.txt]: This looks fine; it's true.

File: molly/molly.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+I do not have blue hair.
```
Comment (embray) [inline molly/molly.txt]: This is fine.

**Pull request merged**